tests/v16/conftest.py

```diff
@@ -1,11 +1,6 @@
-try:
-    from unittest.mock import AsyncMock
-except ImportError:
-    # Python 3.7 and below don't include unittest.mock.AsyncMock. Hence,
-    # we need to resolve to a package on pypi.
-    from asynctest import CoroutineMock as AsyncMock
-
+import sys  
 import pytest
+from unittest.mock import Mock
```

Comment:
The tests rely on `AsyncMock`, not `Mock`. 
```suggestion
from unittest.mock import AsyncMock
```

Reply:
so can i used 
Import AsyncMock
and remove just the unittest.mock

Reply:
I"m trying to be patient and guiding you through required changes step by step.  
But I feel you don't read my comments  and  try to understand them.  

The exact fix is suggested in my comment [here](https://github.com/mobilityhouse/ocpp/pull/566#discussion_r1452546612). Make sure to apply that and the other suggesions. And make sure that the CI tests for  for Python 3.8, 3.9 and 3.10 pass.

Reply:
Sorry I am new to GitHub that's why I don't much know about it.
But I am trying to make it right.